code/modules/clothing/masks/hailer.dm

```diff
@@ -0,0 +1,171 @@
+	flags_inv = HIDEFACE
+	visor_flags = MASKCOVERSMOUTH | BLOCK_GAS_SMOKE_EFFECT | MASKINTERNALS
+	visor_flags_inv = HIDEFACE
+	var/aggressiveness = 2
+	var/cooldown_special
+	var/recent_uses = 0
+
+/obj/item/clothing/mask/gas/sechailer/swat
+	name = "\improper SWAT mask"
+	desc = "A close-fitting tactical mask with an especially aggressive Compli-o-nator 3000."
+	action_button_name = "HALT!"
+	icon_state = "swat"
+	aggressiveness = 3
+	ignore_maskadjust = 1
+
+/obj/item/clothing/mask/gas/sechailer/cyborg
+	name = "security hailer"
+	desc = "A set of recognizable pre-recorded messages for cyborgs to use when apprehending criminals."
+	icon = 'icons/obj/device.dmi'
+	icon_state = "taperecorder_idle"
+	aggressiveness = 1 //Borgs are nicecurity!
+	ignore_maskadjust = 1
+
+/obj/item/clothing/mask/gas/sechailer/cyborg/New()
+	..()
+	verbs -= /obj/item/clothing/mask/gas/sechailer/verb/adjust
+
+
+/obj/item/clothing/mask/gas/sechailer/attackby(obj/item/weapon/W as obj, mob/user as mob, params)
+	if(istype(W, /obj/item/weapon/screwdriver))
+		switch(aggressiveness)
+			if(1)
+				user << "<span class='notice'>You set the restrictor to the middle position.</span>"
+				aggressiveness = 2
+			if(2)
+				user << "<span class='notice'>You set the restrictor to the last position.</span>"
+				aggressiveness = 3
+			if(3)
+				user << "<span class='notice'>You set the restrictor to the first position.</span>"
+				aggressiveness = 1
+			if(4)
+				user << "<span class='danger'>You adjust the restrictor but nothing happens, probably because its broken.</span>"
+	else if(istype(W, /obj/item/weapon/wirecutters))
+		if(aggressiveness != 4)
+			user << "<span class='danger'>You broke it!</span>"
+			aggressiveness = 4
+	else
+		..()
+
+/obj/item/clothing/mask/gas/sechailer/verb/hailer_overload(var/mob/living/user)
+	if(ishuman(user))
+		user.adjust_fire_stacks(15)
+		user.IgniteMob()
+		user.adjustBrainLoss(20)
+		user << "<span class='danger'>\The [src] spews battery acid all over you and explodes.</span>"
+		playsound(src.loc, 'sound/effects/spray.ogg', 10, 1, -3)
+		qdel(src)
+
+/obj/item/clothing/mask/gas/sechailer/cyborg/hailer_overload(var/mob/living/user)
+	empulse(src, 1, 1, 0)
+	user << "<span class='danger'>\The [src]'s power modulator overloads.</span>"
+
+/obj/item/clothing/mask/gas/sechailer/verb/adjust()
+	set category = "Object"
+	set name = "Adjust Mask"
+	adjustmask(usr)
+
+/obj/item/clothing/mask/gas/sechailer/attack_self()
+	halt()
+
+/obj/item/clothing/mask/gas/sechailer/verb/halt()
+	set category = "Object"
+	set name = "HALT"
+	set src in usr
+	if(!istype(usr, /mob/living))
+		return
+	if(!can_use(usr))
+		return
+
+	var/phrase = 0	//selects which phrase to use
+	var/phrase_text = null
+	var/phrase_sound = null
+
+	if(cooldown < world.time - 30) // A cooldown, to stop people being jerks
+		recent_uses++
+		if(cooldown_special < world.time - 150) //A better cooldown that lights jerks on fire
```

Comment:
Spamming the chat is running scripts to ensure the onky thing people see is some stupid bullshit.

This would be a good suit for a Disturbing the Peace crime